sql-plugin/src/main/shim301/scala/com/nvidia/spark/rapids/shims/spark301/RapidsShuffleInternalManager.scala

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2019-2021, NVIDIA CORPORATION.
+ * Copyright (c) 2019-2022, NVIDIA CORPORATION.
```

Comment:
Copyright update on a file whose contents were not modified.  Comment applies to many other files.

Reply:
Are we OK with moving files without changing the copyright? I can revert this change.

Reply:
The copyright is on the contents of the file, not the name of the file.

Reply:
Done